coriolis/providers/backup_writers.py

```diff
@@ -169,6 +169,13 @@ def __init__(self, path, disk_id, compress_transfer=None):
         self._stderr = None
         self._offset = None
         self._ssh = None
+        self._sender_q = eventlet.Queue(maxsize=5)
+        self._enc_q = eventlet.Queue(maxsize=5)
+        self._sender_evt = None
+        self._encoder_evt = None
```

Comment:
NIT: I'd default this to `[]` to prevent possible accidental iterations over `None`

Reply:
Done